index.html

```diff
@@ -11,14 +11,6 @@
      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
 </head>
 <body>
-    <div class="phone-status">
-        <img src="/images/svg/Time Style.svg" alt="time">
-        <div class="left-status">
-            <img src="/images/svg/Cellular Connection.svg" alt="cellular-status">
-            <img src="/images/svg/Wifiwifi.svg" alt="wifi">
-            <img src="/images/svg/Battery.svg" alt="battery">
-        </div>
-    </div>
     <header id="navigation">
         <a href="#" class="logo">my logo</a>
         <nav>
```

Comment:
- [ ] While I can see your hamburger on the dom element, I could not see it on the UI, it must be a styling issue. Kindly dig further so that it would show.

![image](https://user-images.githubusercontent.com/86200356/234816499-ae8b0dc4-94dc-48f2-ab11-7b477fbf9210.png)